distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -6468,7 +6468,7 @@ The Platform Commands are installed when the Platform application is installed.
 
 ----
 ddf@local>platform:
-platform:describe    platform:envlist
+platform:configstatus    platform:describe    platform:envlist
```

Comment:
Should we prefix all of our commands with `config-`, e.g., `config-status`, `config-export`, etc?